site/src/utils/richParameters.ts

```diff
@@ -38,7 +41,7 @@ const isValidValue = (
   return true;
 };
 
-export const useValidationSchemaForRichParameters = (
+export const validateRichParameters = (
```

Comment:
Updated the name because it felt like it could mislead someone into thinking it's a React hook